### PR TITLE
Disabling argument parsing on flux command

### PR DESCRIPTION
**Pull request by luizbafilho**
We no longer need to use `--` after a flux command.

### CODE REVIEW

File: cmd/wego/flux/cmd.go [hunk 1]
```diff
@@ -11,10 +11,11 @@ import (
 )
 
 var Cmd = &cobra.Command{
-	Use:     "flux -- [flux commands or flags]",
-	Short:   "Use flux commands",
-	Example: "wego flux -- install -h",
-	Run:     runCmd,
+	Use:                "flux [flux commands or flags]",
+	Short:              "Use flux commands",
+	DisableFlagParsing: true,
```
Comment (josecordaz) [inline cmd/wego/flux/cmd.go:16]: Nice finding

**Review by josecordaz — APPROVED** [review]
LGTM

**Comment by luizbafilho** [comment]
@J-Thompson12 look what got it working. 🎉

File: cmd/wego/flux/cmd.go (hunk 1 above)
Comment (luizbafilho) [reply cmd/wego/flux/cmd.go:16]: hahah yep, was looking for something else and I looked to that and had aha 💡 moment.

**Pull request merged**